### PR TITLE
[pip][design] PIP-276: Add metric `pulsar_topic_load_times`

### PR DESCRIPTION
**Pull request by Technoboy-**
Fixes #xyz

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by coderzc** [comment]
@Technoboy- Please modify the PIP number, PIP-274 already exists

File: pip/pip-274.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Motivation
+
+The metrics are all started with `pulsar_`, so that both users and operators can quickly find the metrics of the entire system through this prefix. However, due to some other reasons, 
+it was found that `topic_load_times` was missing the prefix, so want to get it right.
+
+# Goals
+
+## In Scope
+
+### Proposed changes
```
Comment (asafm) [inline pip/pip-274.md:10]: Could you move this to High Level Design section? This is not a goal

File: pip/pip-274.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Design & Implementation Details
+
+
+
+## Public-facing Changes
+
+
+
+### Public API
```
Comment (asafm) [inline pip/pip-274.md:36]: Feel free to delete any title with empty body

File: pip/pip-274.md (hunk 1 above)
Comment (Technoboy-) [reply pip/pip-274.md]: yes, thanks for reviewing. Addressed

File: pip/pip-274.md (hunk 2 above)
Comment (Technoboy-) [reply pip/pip-274.md]: fixed

**Review by asafm — COMMENTED** [review]
Looks good now

**Review by poorbarcode — APPROVED** [review]

**Review by mattisonchao — APPROVED** [review]

**Comment by asafm** [comment]
@Technoboy- You merged it too soon - you only have 2 +1 binding votes, no?

File: pip/pip-274.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Motivation
+
```
Comment (StevenLuMT) [inline pip/pip-274.md:2]: pip‘s name is [pip][design] **PIP-276**: Add metric pulsar_topic_load_times
but file name is pip/**pip-274.md** ?

Comment (asafm) [reply pip/pip-274.md:2]: Good call!! @Technoboy- please note

Comment (Technoboy-) [reply pip/pip-274.md:2]: yes, fixed this in https://github.com/apache/pulsar/pull/20614

**Pull request merged**